scene/2d/line_2d.cpp

```diff
@@ -48,6 +48,32 @@ Line2D::Line2D() :
 	_round_precision = 8;
 }
 
+Rect2 Line2D::_edit_get_rect() const {
```

Comment:
I'm pretty sure you cannot do that without modifying the _edit_set_rect function. I would leave it as is for now.

Reply:
Leaving it as it is means it's not selectable. The standard transform-based `Node2D::_edit_set_rect()` works fine in all these cases.